tests/e2e/scenarios/ha/README.md

```diff
@@ -0,0 +1,105 @@
+# Skupper HA (High Availability) Test
+
+This README provides an overview of the "Skupper HA Test". 
+
+## Overview
+
+This test creates two Skupper sites with the HA feature enabled, this creates an additional instance of the skupper router in each skupper site.
+A backend application is deployed in the east_ha namespace and exposed via Skupper in the west_ha namespace.
+Once all skupper components are up, a k8s job is created using the Locust image to send a lot of HTTP POST requests to the backend endpoint.
+While the locust job is running, we start killing the router pods, first the router1, then the router2, in a looping.
+When the locust job ends, we get its logs and inspect the number of failures.
+
+## Prerequisites
+
+- Two Kubernetes clusters (identified as "west" and "east" in your inventory)
+- Ansible with the following collections installed:
+  - `e2e.tests`
```

Comment:
Is this the internal collection under `./tests/`?

Reply:
Yes, it is under tests